src/Text/Pandoc/Readers/ODT/ContentReader.hs

```diff
@@ -75,6 +75,10 @@ data ReaderState
                  , styleTrace       :: [Style]
                    -- | Keeps track of the current depth in nested lists
                  , currentListLevel :: ListLevel
+                   -- | Keeps track of the previous list start counters,
+                   -- so whenever a new list wants to continue numbering,
+                   -- we know what number to start from.
+                 , listContinuationStartCounters :: M.Map ListLevel Int
```

Comment:
I think this comment may be a bit misleading ; isn't this keeping track of what the *next* start number should be if another list in the same level is marked as continuing?

Reply:
Will improve.